### PR TITLE
Fix stray settings item separator tr

### PR DESCRIPTION
**Pull request by doubleO8**
Fix two <tr> tags not having a table as parent

### CODE REVIEW

**Review by zagrim — DISMISSED** [review]
Good catch, looks good to me. Maybe @ccrisan could merge this?

**Comment by MichaIng** [comment]
I'm not sure whether this is actually intended to separate general settings from camera/streaming related settings. But IMO these additional/doubled separators between the settings blocks do not look nice.

I applied your change, but then there is an ugly half-length line within "Preferences" and "General Settings". Such a separator line only makes sense within one settings bock as a second separation layer. So I suggest so imply remove them.

File: motioneye/templates/main.html [hunk 1]
```diff
@@ -136,11 +136,11 @@
                             <td class="settings-item-value"><input type="text" class="range styled prefs" id="resolutionDimmerSlider"></td>
                             <td><span class="help-mark" title="dims the actual resolution of all cameras to save network bandwidth and traffic (doesn't work on simple MJPEG cameras)">?</span></td>
                         </tr>
+                        <tr class="settings-item">
+                            <td colspan="100"><div class="settings-item-separator" style="margin: 0px 0px 1.5em 0px;"></div></td>
+                        </tr>
```
Comment (MichaIng) [inline motioneye/templates/main.html:141]:
```suggestion
```

File: motioneye/templates/main.html [hunk 2]
```diff
@@ -232,13 +232,13 @@
...
                     </table>
                     {% endif %}
                     {% endfor %}
 
-                    <tr class="settings-item">
-                        <td colspan="100"><div class="settings-item-separator" style="margin: 0px 0px 1.5em 0px;"></div></td>
-                    </tr>
```
Comment (MichaIng) [inline motioneye/templates/main.html:242]:
```suggestion
```
Removing doubled empty line.

File: motioneye/templates/main.html [hunk 3]
```diff
@@ -136,11 +136,11 @@
...
                             <td><span class="help-mark" title="dims the actual resolution of all cameras to save network bandwidth and traffic (doesn't work on simple MJPEG cameras)">?</span></td>
                         </tr>
+                        <tr class="settings-item">
+                            <td colspan="100"><div class="settings-item-separator" style="margin: 0px 0px 1.5em 0px;"></div></td>
+                        </tr>
                     </table>
 
-                    <tr class="settings-item">
-                        <td colspan="100"><div class="settings-item-separator" style="margin: 0px 0px 1.5em 0px;"></div></td>
-                    </tr>
```
Comment (MichaIng) [inline motioneye/templates/main.html:144]:
```suggestion
```
Removing doubled empty line.

Comment (cclauss) [reply motioneye/templates/main.html:141]: FYI: If we were to turn on https://pre-commit.ci/ then these corrections would be made automatically.

Comment (cclauss) [reply motioneye/templates/main.html:141]: FYI: If we were to turn on https://pre-commit.ci/ then these corrections could be made automatically.

Comment (MichaIng) [reply motioneye/templates/main.html:141]: Can be also done with other tools via GitHub actions, I guess also with a dedicated pre-commit workflow which triggers a PR as GitHub bot when a diff was found. Generally for now I think we should stay with GitHub actions and not switch to/include further external CI/CD providers, just to keep it transparent and simple to connect workflows with each other at some point.

**Review by MichaIng — APPROVED** [review]
Rebased and resolved conflicts, all check green, LGTM 👍.

**Pull request merged**